as2_utilities/as2_external_object_to_tf/package.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>as2_external_object_to_tf</name>
+  <version>0.0.1</version>
+  <description>Adds external objects pose to tf</description>
+  <maintainer email="javier.mdeza@alumnos.upm.es">Javier Melero Deza</maintainer>
```

Comment:
Maintainer should we <maintainer email="cvar.upm3@gmail.com">CVAR-UPM</maintainer>
You should be author

Reply:
Done